src/pyobo/sources/gtdb.py

```diff
@@ -96,7 +96,10 @@ def _process_row(tax_string, ncbitaxon_id) -> Iterable[Term]:
         if ncbitaxon_id and level == "s":
             # if the level is "s", it's a species. There might be multiple
             # mappings to NCBITaxon, so we only use "see also" as the predicate
-            term.append_xref(Reference(prefix="ncbitaxon", identifier=ncbitaxon_id))
+            term.append_xref(
+                Reference(prefix="ncbitaxon", identifier=ncbitaxon_id),
+                # TODO @jose use confidence=... keyword here
```

Comment:
@jplfaria this is the place where you can write the confidence. If there are some missing SSSOM-like fields you want to use and I can add them in a follow-up PR

```python
term.append_xref(
                Reference(prefix="ncbitaxon", identifier=ncbitaxon_id),
                confidence=0.99, # or a variable
```

Reply:
Thanks I will let you know if I need anything else!